ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/helpers/AndroidApplicationContext.kt

```diff
@@ -0,0 +1,20 @@
+package com.revenuecat.purchases.ui.revenuecatui.helpers
+
+import android.content.Context
+
+/**
+ * Abstraction around [Context]
+ */
+internal interface ApplicationContext {
+    fun getApplicationName(): String
+}
+
+internal class AndroidApplicationContext(private val applicationContext: Context) : ApplicationContext {
+    override fun getApplicationName(): String {
+        return applicationContext.applicationInfo.loadLabel(applicationContext.packageManager).toString()
+    }
+}
+
+internal fun Context.toAndroidContext(): ApplicationContext {
+    return AndroidApplicationContext(this)
```

Comment:
Might be a good idea to call `applicationContext` instead of `this`. That way we make sure people don't pass a non application context here. It should be fine to call application context on the applicationContext afaik

Reply:
👍🏻  done!